source/includes/administration/_pricings.md

```diff
@@ -238,6 +238,21 @@ Required | &nbsp;
 `cogs`<br/>*Map* | Map from currency code to cogs
 `unitPrice`<br/>*Map* | Map from currency code to unit price
 
+Optional | &nbsp;
+---- | -----------
+`pricingTiers` <br/>*Array<PricingTier>* | List of pricing tiers on priced product
+
+Pricing tier
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | UUID of the pricing tier. (Auto Generated)
+`pricingMode`<br/>*Enum* | The pricing mode of the pricing tier. Must be one of "FLAT_FEE" or "PER_UNIT".
+`lowerBound`<br/>*double* | The pricing tier's lower bound.
+`upperBound`<br/>*double* | The pricing tier's upper bound. A null value indicates that there is no upper bound for the given tier.
+`price`<br/>*Map<string, double* | The pricing tier's price in each of the pricing product's supported currencies.
```

Comment:
This should be an `Object`. What are the keys?

Reply:
this should be pricingTier.id, pricingTier.pricingMode, etc

Reply:
Does this need to be a `PricingTier` or another object?

Reply:
it's a list of objects, they aren't named

Reply:
@m-orfanos did you get your original question answered? If not, I'm not sure what you're asking 🤔

Reply:
This can't be defined as a `Map`. The API uses JSON to send back and forth. `price` should be defined as an `Object`, what are the keys? They should be documented.

Reply:
```
   "price": {
                "CAD": 20,
                "USD": 17
              },
```

Reply:
This would be clearer as a list of objects imo

```
{
  "price": [
    {"currency": "CAD", "value": 20},
    {"currency": "USD", "value": 17},
  ]
}
```

Reply:
you're suggesting we completely change the api to this?^

Reply:
no, at first glance it looked odd, but I think it's fine since it's currency code

Reply:
can you use similar wording to the others 🙏

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/403/commits/bc146fc31b56590a678f6ae59e7320d19ec995b1